src/app/Components/ProgressiveOnboarding/ProgressiveOnboardingFindSavedArtwork.tsx

```diff
@@ -0,0 +1,46 @@
+import { Flex, Popover, Text } from "@artsy/palette-mobile"
+import { BottomTabType } from "app/Scenes/BottomTabs/BottomTabType"
+import { GlobalStore } from "app/store/GlobalStore"
+
+type ProgressiveOnboardingFindSavedArtworkProps = {
+  tab: BottomTabType
+}
+
+export const ProgressiveOnboardingFindSavedArtwork: React.FC<
+  ProgressiveOnboardingFindSavedArtworkProps
+> = ({ children, tab }) => {
+  const { isDismissed: _isDismissed } = GlobalStore.useAppState(
+    (state) => state.progressiveOnboarding
+  )
+  const { dismiss } = GlobalStore.actions.progressiveOnboarding
+  const profileTabProps = GlobalStore.useAppState(
+    (state) => state.bottomTabs.sessionState.tabProps.profile
+  )
+
+  const isDismissed = _isDismissed("find-saved-artwork").status
+
+  const isFindSavedArtworkDisplayable = !isDismissed && !!profileTabProps?.savedArtwork
```

Comment:
I find the naming around `_isDismissed` a bit confusing. Maybe we could avoid the renaming like this?

```suggestion
  const isFindSavedArtworkDisplayable = !isDismissed("find-saved-artwork").status && !!profileTabProps?.savedArtwork
```

Reply:
makes sense 👍 
Added the changes here https://github.com/artsy/eigen/pull/9224/commits/a8798b0fe53c84dd9fee41b0adc347d1c8fa4874 because was missing one change from the suggestion